airbyte-integrations/connectors/destination-bigquery/src/test-integration/java/io/airbyte/integrations/destination/bigquery/BigQueryDestinationTest.java

```diff
@@ -227,12 +227,12 @@ void testCheckSuccess(final DatasetIdResetter resetDatasetId) {
   void testCheckFailure(final DatasetIdResetter resetDatasetId) {
     ((ObjectNode) config).put(BigQueryConsts.CONFIG_PROJECT_ID, "fake");
     resetDatasetId.accept(config);
-    final AirbyteConnectionStatus actual = new BigQueryDestination().check(config);
```

Comment:
I am not seeing any functional changes here. If the gist of this change is renaming `actual` to `status` then lets keep it out of this PR to simplify the job of reviewing it

Reply:
I'll revert these changes, as mentioned in the other comment it was primarily a byproduct of translating the previous tests from GL where some of the variables were `var actual = destination.check(config)` to use less data inference variables and agree they should not be in the review